model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -78,31 +78,45 @@ message Components {
 // A Pipeline is a hierarchical graph of PTransforms, linked
 // by PCollections.
 //
-// This is represented by a number of by-reference maps to nodes,
-// PCollections, SDK environments, UDF, etc., for
-// supporting compact reuse and arbitrary graph structure.
+//   Impulse -> PCollection -> ParDo -> PCollection -> GroupByKey -> ...
+//                                   \> PCollection -> ParDo      -> ...
```

Comment:
Is this correct? I thought it would look like:

```
Impulse -> PCollection -> ParDo -> PCollection -> GroupByKey -> ...
                                               \> ParDo      -> ...
```

Or maybe the ParDo you're documenting is a multi-output one?

Reply:
Its a multi output one.